### PR TITLE
Mark entrypoint as required for manual function creation

### PR DESCRIPTION
**Pull request by vermakhushboo**
## What does this PR do?

Mark entrypoint as required for manual function creation because when user creates a function from console, the first deployments gets created automatically. If they don't specify entrypoint in function creation, it would throw server error.

## Test Plan

Tested manually

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Review by ArmanNik — APPROVED** [review]

**Review by christyjacob4 — APPROVED** [review]
Lets also wait for a review from the frontend team

**Pull request merged**
